### ISSUE TITLE
Containers tab is shown for non-thin-edge devices

### ISSUE THREAD

**Issue by zhong-ys** [thin-edge/tedge-container-plugin#51]
I have thick edge on k8s instance and it is connected to a cloud tenant so I can manage the software of the thick edge remotely. The UI extension of this wedge-container-plugin somehow recognised Thick edge as a thin-edge device on the cloud tenant and shows themselves up there. Is it possible to disable this for Thick edge device, as even the thick edge has a thin-edge inside, it dosen't have tedge-container-plugin installed. 

Cloud instance: eu-latest 
UI Extension version 1.0.2
Edge on k8s: 18.0.1

**Comment by zhong-ys** [comment]
<img width="923" alt="Screenshot 2024-10-22 at 13 53 46" src="https://github.com/user-attachments/assets/a577745d-8e97-479d-96f8-643d6c9a5395">
In the screenshot is a thick edge device registered on a cloud tenant. The Containers and Container Groups should not be shown here.

**Comment by reubenmiller** [comment]
Do either of the tabs have items under it?

**Comment by reubenmiller** [comment]
Looking at the UI code, those tabs should only appear if the following Guard is active, https://github.com/thin-edge/tedge-container-plugin/blob/main/ui/src/shared/container.guard.ts

So I would assume that it is visible because it has at least one container or container-group child...then the real question is "why does it have a container or container-group item?"...maybe something is publishing to the wrong MQTT broker and it is hitting the MQTT broker used by the thick-edge agent (which is a thin-edge.io instance).

**Comment by zhongys-c8y** [comment]
> Do either of the tabs have items under it?

No, both were empty

**Comment by reubenmiller** [comment]
> Checked both are empty. But Services tab has a kedge-agent with 'container' type which should be thin-edge itself, right? I guess this is the reason.

Yes that looks to be the reason (the fact that the kedge-agent uses the `container` type)...so it is a false positive. We could extend the Guard to also check for the presence of the `container` fragment.

**Comment by reubenmiller** [comment]
For example, the following query should hopefully be enough to remove the false positive.

```sh
(serviceType eq 'container' or serviceType eq 'container-group') and has(container)
```

**Comment by zhongys-c8y** [comment]
That would be great. As thick edge dosen't officially have this remote container monitoring feature, not showing those two tabs can avoid confusion I think. Thank you!

**Comment by reubenmiller** [comment]
@zhong-ys It should be fixed in the latest ui plugin, 1.0.3 (part of the [1.3.2)](https://github.com/thin-edge/tedge-container-plugin/releases/tag/1.3.2) release)

**Comment by zhongys-c8y** [comment]
Thank you!
